<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenPSTD</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="OpenPSTD::GUI::Viewer2D" name="mainView" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Frame</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollBar" name="hsbFrame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExport_HDF5"/>
     <addaction name="actionExport_Image"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Geometry"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuDimension">
     <property name="title">
      <string>Dimension</string>
     </property>
     <addaction name="action2D"/>
     <addaction name="action3D"/>
    </widget>
    <addaction name="menuDimension"/>
    <addaction name="actionDocument_Settings"/>
    <addaction name="actionApplication_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpenPSTD_help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_icons"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMove_scene"/>
    <addaction name="actionView_complete_scene"/>
    <addaction name="actionShow_grid"/>
   </widget>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="actionAdd_Receiver_Speaker"/>
    <addaction name="actionDelete_selected"/>
    <addaction name="actionResize_domain"/>
    <addaction name="actionEdit_properties_of_domain"/>
    <addaction name="separator"/>
    <addaction name="actionStart_simulation"/>
    <addaction name="actionStopAction"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Mouse Tools</string>
    </property>
    <addaction name="actionSelect"/>
    <addaction name="actionMove_scene"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Domain"/>
    <addaction name="actionAdd_speaker"/>
    <addaction name="actionAdd_Receiver"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOption"/>
   <addaction name="menuTool"/>
   <addaction name="menuOperations"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionMove_scene"/>
   <addaction name="actionAdd_Domain"/>
   <addaction name="actionAdd_speaker"/>
   <addaction name="actionAdd_Receiver"/>
   <addaction name="separator"/>
   <addaction name="actionView_complete_scene"/>
   <addaction name="separator"/>
   <addaction name="actionStopAction"/>
   <addaction name="actionStart_simulation"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action2D">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2D</string>
   </property>
  </action>
  <action name="action3D">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>3D</string>
   </property>
  </action>
  <action name="actionOpenPSTD_help">
   <property name="text">
    <string>OpenPSTD help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionMove_scene">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/noun_1476.png</normaloff>:/Images/noun_1476.png</iconset>
   </property>
   <property name="text">
    <string>Move scene</string>
   </property>
   <property name="toolTip">
    <string>Move the view</string>
   </property>
  </action>
  <action name="actionView_complete_scene">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/noun_70430_cc.png</normaloff>:/Images/noun_70430_cc.png</iconset>
   </property>
   <property name="text">
    <string>View complete scene</string>
   </property>
  </action>
  <action name="actionAdd_Domain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/noun_40980_cc.png</normaloff>:/Images/noun_40980_cc.png</iconset>
   </property>
   <property name="text">
    <string>Add Domain</string>
   </property>
  </action>
  <action name="actionShow_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show grid</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/noun_120460_cc.png</normaloff>:/Images/noun_120460_cc.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionDelete_selected">
   <property name="text">
    <string>Delete selected</string>
   </property>
   <property name="toolTip">
    <string>Delete selected</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionResize_domain">
   <property name="text">
    <string>Resize domain</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEdit_properties_of_domain">
   <property name="text">
    <string>Edit Subdomain Boundary Conditions</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDocument_Settings">
   <property name="text">
    <string>Calculation Settings</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionAdd_speaker">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/noun_100_cc.png</normaloff>:/Images/noun_100_cc.png</iconset>
   </property>
   <property name="text">
    <string>Add Speaker</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionAdd_Receiver">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/noun_33232_cc.png</normaloff>:/Images/noun_33232_cc.png</iconset>
   </property>
   <property name="text">
    <string>Add Receiver</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionAbout_icons">
   <property name="text">
    <string>About icons</string>
   </property>
  </action>
  <action name="actionStart_simulation">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Play.png</normaloff>:/Images/Play.png</iconset>
   </property>
   <property name="text">
    <string>Start simulation</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionApplication_Settings">
   <property name="text">
    <string>Application Settings</string>
   </property>
  </action>
  <action name="actionExport_HDF5">
   <property name="text">
    <string>HDF5</string>
   </property>
  </action>
  <action name="actionExport_Image">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionStopAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Stop.png</normaloff>:/Images/Stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop current action</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="actionAdd_Receiver_Speaker">
   <property name="text">
    <string>Add Receiver(s) or Speaker(s)</string>
   </property>
  </action>
  <action name="actionImport_Geometry">
   <property name="text">
    <string>Import Geometry</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenPSTD::GUI::Viewer2D</class>
   <extends>QWidget</extends>
   <header>GUI/Viewer2D.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
